content/docs/next-release/operator.md

```diff
@@ -391,6 +391,22 @@ storage:
     image: jaegertracing/jaeger-es-index-cleaner  // image of the job
 ```
 
+## Spark dependencies
+
+Spark dependencies collects spans from storage, analyzes links between services and stores them for later presentation in the UI.
+This job is needed only with production strategy and it supports Cassandra and Elasticsearch backends.
```

Comment:
"This job can only be used with the `production` strategy and storage type `cassandra` or `elasticsearch`."?